packages/core/src/progress/LinearProgress/LinearProgress.tsx

```diff
@@ -43,8 +45,8 @@ export const LinearProgress = forwardRef<HTMLDivElement, LinearProgressProps>(
       hideLabel = false,
       max = 100,
       min = 0,
-      value = 0,
```

Comment:
is this considered a breaking change ?
Should we  leave this as 0 and opt into indeterminate ?
At some point a breaking change release we could make this the default ?